firefox-ios/nimbus-features/onboardingFrameworkFeature.yaml

```diff
@@ -10,6 +10,11 @@ features:
           Toggle between legacy (false) and modern (true) UI variants.
         type: Boolean
         default: false
+      isJapan:
```

Comment:
I feel like this naming could be improved. Maybe something like "should-use-japan-assets" because reading `isJapan` without context can be confusing